production-docker-compose.yml

```diff
@@ -13,9 +13,9 @@ services:
     environment:
       - PROJECT_NAME
       - DEBUG=False
-      - POSTGRES_USER
-      - POSTGRES_NAME
-      - POSTGRES_HOST
-      - POSTGRES_PORT
-      - POSTGRES_PASSWORD
-      - DJANGO_SECRET_KEY
+      - POSTGRES_USER=${PRODUCTION_POSTGRES_USER}
```

Comment:
I would prefer we use the non-prefixed env vars for the production side (i.e. `POSTGRES_USER` not `PRODUCTION_POSTGRES_USER`)